Content.Server/Tabletop/TabletopBackgammonSetup.cs

```diff
@@ -0,0 +1,91 @@
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Server.Tabletop
+{
+    [UsedImplicitly]
+    public class TabletopBackgammonSetup : TabletopSetup
+    {
+        [DataField("boardPrototype")]
+        public string BackgammonBoardPrototype { get; } = "BackgammonBoardTabletop";
+
+        [DataField("whitePiecePrototype")]
+        public string WhitePiecePrototype { get; } = "WhiteTabletopPiece";
+
+        [DataField("blackPiecePrototype")]
+        public string BlackPiecePrototype { get; } = "BlackTabletopPiece";
+
+
+        public override void SetupTabletop(TabletopSession session, IEntityManager entityManager)
+        {
+            var board = entityManager.SpawnEntity(BackgammonBoardPrototype, session.Position);
+
+            const float borderLengthX = 7.35f; //BORDER
+            const float borderLengthY = 5.60f; //BORDER
+
+            const float boardDistanceX = 1.25f;
+            const float pieceDistanceY = 0.80f;
+
+            float getXposition (float distanceFromSide, bool isLeftSide)
+            {
+                return isLeftSide ? -borderLengthX + (distanceFromSide * boardDistanceX)
+                : borderLengthX - (distanceFromSide * boardDistanceX);
```

Comment:
Just store `borderLengthX - (distanceFromSide * boardDistanceX);` in a variable named `pos` or something, and then do `return isLeftSide ? -pos : pos`. Same goes for Y position function.

Reply:
That is a lot more elegant thank you.